client/jetpack-cloud/sections/pricing/header/index.tsx

```diff
@@ -5,7 +5,7 @@ import { preventWidows } from 'calypso/lib/formatting';
 
 import './style.scss';
 
-const Header: React.FC< Props > = () => {
+const Header: React.FC< Props > = ( { title }: Props ) => {
```

Comment:
You can just use `( { title } )` as it is already specified of type Props.

Reply:
Thanks for spotting this Andrii. done on removing it 👍